src/app/pages/ParatimeDashboardPage/index.tsx

```diff
@@ -21,14 +21,19 @@ export const ParatimeDashboardPage: FC = () => {
       <Divider variant="layout" sx={{ mt: isMobile ? 4 : 0 }} />
       <LatestTransactions />
       <Grid container spacing={4}>
-        <Grid item xs={12} md={6} sx={{ display: 'flex', order: isMobile ? 1 : 0 }}>
+        <Grid item xs={12} md={6} sx={{ display: 'flex', order: isMobile ? 2 : 0 }}>
```

Comment:
This change is not needed, right?

Reply:
I'm not sure. I assumed that now (on mobile) we want it at the 3rd spot, after Blocks and Tokens. Isn't that the case?

Reply:
Right, undone.